stats/thresholds.go

```diff
@@ -44,18 +44,18 @@ type Threshold struct {
 	parsed *thresholdExpression
 }
 
-func newThreshold(src string, abortOnFail bool, gracePeriod types.NullDuration) (*Threshold, error) {
+func newThreshold(src string, abortOnFail bool, gracePeriod types.NullDuration) *Threshold {
 	parsedExpression, err := parseThresholdExpression(src)
 	if err != nil {
-		return nil, err
+		return nil
 	}
```

Comment:
You need to remove this lines otherwise this will return a nil an then you will get nullPointerException

Reply:
Good catch Indeed 👀  I actually ran into the problem while writing test for `Parse` 👍🏻